packages/react-notion-x/src/third-party/collection-view-board.tsx

```diff
@@ -109,8 +109,8 @@ function Board({ collectionView, collectionData, collection, padding }) {
                       />
                     ) : (
                       <span>
-                        <EmptyIcon className='notion-board-th-empty' /> No
-                        Select
+                        <EmptyIcon className='notion-board-th-empty' />
+                        {`No ${schema.name}` || 'No Select'}
```

Comment:
Here you are missing the check for `schema.name`, what this will render if schema.name is not available?

Your provided `No Select` optional fallback does not work here.

Reply:
agreed

Reply:
Thanks @s4nju and @transitive-bullshit, it should be fixed now.